crates/common_utils/src/types.rs

```diff
@@ -582,6 +582,12 @@ impl StringMajorUnit {
     }
 }
 
+impl Display for StringMajorUnit {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
```

Comment:
I'm not sure if we'd want to implement `Display` here, or if we're to use one of the methods already available.

CC: @jarnura @sahkal